File: LR/LR_1-2/5.py
# Ввод данных
x = int(input("Введите X: "))
y = int(input("Введите Y: "))
k = int(input("Введите K: "))

# Проверка, достаточно ли меда
if (x * y) >= k:
    print('YES')
else:
    print('NO')

File: LR/LR_1-2/5.pas
program HoneyCounterLol;

var x,y,k:integer;

begin
  // Ввод данных
  writeln('Введите X, Y и K');
  readln(x, y, k);
  
  // Проверка, достаточно ли меда
  if (x * y) >= k then
    begin
    writeln('YES');
    end
  else
    writeln('NO');
end.